biome.json

```diff
@@ -14,7 +14,8 @@
             "src/protocol/types.ts",
             "target",
             "docs/static",
-            "docs/.docusaurus"
+            "docs/.docusaurus",
+            "themes"
```

Comment:
I think the theme JSON files should be formatted so that in the future if we update the theme files, we won't have to commit formatting-only changes.